programs/svm-spoke/src/instructions/fill.rs

```diff
@@ -79,18 +79,14 @@ pub struct FillV3Relay<'info> {
     pub system_program: Program<'info, System>,
 }
 
-pub fn fill_v3_relay<'info>(
+pub fn fill_relay<'info>(
     ctx: Context<'_, '_, '_, 'info, FillV3Relay<'info>>,
     relay_data: Option<V3RelayData>,
     repayment_chain_id: Option<u64>,
     repayment_address: Option<Pubkey>,
 ) -> Result<()> {
-    let FillV3RelayParams { relay_data, repayment_chain_id, repayment_address } = unwrap_fill_v3_relay_params(
-        relay_data,
-        repayment_chain_id,
-        repayment_address,
-        &ctx.accounts.instruction_params,
-    );
+    let FillV3RelayParams { relay_data, repayment_chain_id, repayment_address } =
+        unwrap_fill_relay_params(relay_data, repayment_chain_id, repayment_address, &ctx.accounts.instruction_params);
```

Comment:
This seems bit inconsistent that we rename the internal function to `unwrap_fill_relay_params`, but keep the same `FillV3RelayParams` struct name when deconstructing its parameters.